app/views/news_items/index.html.erb

```diff
@@ -17,7 +17,7 @@
         </h2>
 
         <div class="tariff-markdown">
-          <%= govspeak news_item.paragraphs.first %>
+          <%= govspeak replace_service_tags news_item.paragraphs.first %>
```

Comment:
maybe we can use ( ) here, just for readability:
`govspeak replace_service_tags(news_item.paragraphs.first)`
VS
`govspeak replace_service_tags news_item.paragraphs.first`

Reply:
sure - will look at that

Reply:
done